### PR TITLE
Folder structure and alpha backgrounds

### PR DESCRIPTION
**Pull request by motabass**
implements #163 and #16. I will rebase this when #158 is merged

### CODE REVIEW

**Comment by motabass** [comment]
There is now also multiple fixes for alpha stuff in this PR, so wi would suggest to merge it before 7.2.0

**Comment by tiaguinho** [comment]
Hi @motabass, can you do a rebase?

**Comment by motabass** [comment]
This is done for the moment

**Comment by tiaguinho** [comment]
@motabass this PR is ready for review?

**Comment by motabass** [comment]
It is ready. But with secondary entry-points only.

**Review by tiaguinho — CHANGES_REQUESTED** [review]

File: projects/material-community-components/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+After building your library with `ng build material-community-components`, go to the dist folder `cd dist/material-community-components` and run `npm publish`.
+
+## Running unit tests
+
+Run `ng test material-community-components` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+## Further help
+
+To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
```
[the remark this answers is not in the document]
Comment (motabass) [reply projects/material-community-components/README.md]: Hi @tiaguinho . I made the changes

**Comment by motabass** [comment]
Due to having a Breaking Change, namely only secondary entry points, and many other UI fixes i would really go for version 8.0.0 or maybe even 9.0.0 for the next release. That could also indicate, that that the lib only works from angular 9 upwards (as stated here #168)

What do you think @tiaguinho?

**Comment by tiaguinho** [comment]
I like your idea @motabass.
Let's call the next release 9.0.0.

**Review by tiaguinho — APPROVED** [review]

**Pull request merged**